### PR TITLE
Include note about the requirement of Ruby and compass gem

### PR DESCRIPTION
**Pull request by pisani**
compass (Compass Stylesheet Authoring Framework) commands are used by this generator, however, there where no instructions on how to install compass. This tool does seems to be only be available with the installation of Ruby.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -17,6 +17,13 @@ Install `yo`, `grunt-cli`, `bower`, `generator-angular` and `generator-karma`:
 npm install -g grunt-cli bower yo generator-karma generator-angular
 ```
 
+Install Ruby (required in order to install compass which is used by this generator)
```
Comment (eddiemonge) [inline readme.md]: its only required if you select sass

**Comment by eddiemonge** [comment]
Commit message doesn't follow guidelines

File: readme.md (hunk 1 above)
Comment (pisani) [inline readme.md]: Ok... but new users following the existing instructions (which don't document how to deselect sass) won't get their application launched until Ruby+compass is installed.   Are you suggesting that the mention of Ruby+compass installation be merely re-worded, or that it does not need to be raised at all ?

Comment (eddiemonge) [inline readme.md]: re-worded but hopefully soon removed since ruby-sass will be replaced with node-sass

**Pull request closed without merging**